dist/linux/appimage.sh

```diff
@@ -34,9 +34,10 @@ chmod +x AppDir/usr/bin/Cemu
 
 cp /usr/lib/x86_64-linux-gnu/{libsepol.so.1,libffi.so.7,libpcre.so.3,libGLU.so.1,libthai.so.0} AppDir/usr/lib
 
-export UPD_INFO="gh-releases-zsync|cemu-project|Cemu|ci|Cemu.AppImage.zsync"
+export UPD_INFO="gh-releases-zsync|$GITHUB_REPOSITORY_OWNER|Cemu|latest|Cemu-*x86_64.AppImage.zsync"
```

Comment:
The author specifically recommends _against_ using `latest` and suggests using continuous or just using version tags.

Reply:
Are you sure? Because I had this issue: https://github.com/AppImageCommunity/AppImageUpdate/issues/233

I originally tried to get it working but couldn't because Cemu used to release as pre-releases with varying tag names, I tried continuous, ci, testing, etc and none worked.

Now Cemu is going to make more stable releases instead of making pre-releases, so I rebased my changes and made this PR.